.github/workflows/acceptance.yml

```diff
@@ -195,6 +195,7 @@ jobs:
           run: |
             mkdir -p ${{ steps.config.outputs.acceptance-dir }}
             mkdir -p ${{ steps.config.outputs.acceptance-dir }}-mobile
+            [ "$GITHUB_REF" != "refs/heads/master" ] || export PYTEST_SENTRY_ALWAYS_REPORT=1
```

Comment:
```suggestion
            [ "$GITHUB_REF" = "refs/heads/master" ] && export PYTEST_SENTRY_ALWAYS_REPORT=1
```

More readable.

Reply:
josh, this expression prob evaluates to false on PRs and makes the build fail then

Reply:
Hmm the job seems to continue/run